solutions/fully-configurable/outputs.tf

```
@@ -36,3 +36,28 @@ output "dashboard_url" {
   description = "The dashboard URL of the Watson Discovery instance."
   value       = module.watson_discovery.dashboard_url
 }
+
+output "next_steps_text" {
+  value       = "Now, you can launch the Watson Discovery tool to create a project or manage your sample collections."
+  description = "Next steps text"
+}
+
+output "next_step_primary_label" {
+  value       = "Go to Watson Discovery service dashboard"
+  description = "Primary label"
+}
+
+output "next_step_primary_url" {
+  value       = "${module.watson_discovery.dashboard_url}?paneId=manage"
+  description = "Primary URL"
+}
+
+output "next_step_secondary_label" {
+  value       = "Learn more about Watson Discovery"
+  description = "Secondary label"
+}
+
+output "next_step_secondary_url" {
+  value       = "https://cloud.ibm.com/docs/discovery-data?topic=discovery-data-getting-started"
```

```suggestion
  value       = "https://cloud.ibm.com/docs/discovery-data?topic=discovery-data-about"
```
The link you have provided is a tutorial and this seems more relevant.